telemetry/csharp/ToolkitTelemetryGenerator.Tests/sampleDefinitions.json

```diff
@@ -0,0 +1,77 @@
+{
+    "types": [
```

Comment:
at some point we should probably share our test definitions across generators so that when we add new features - we ensure it's caught in all places.